js/app.js

```diff
@@ -1,205 +0,0 @@
-
-$(document).ready(function () {
-  if (!storageStatus('localStorage'))
-    $('.storageErrorStatus')[0].style.visibility = 'visible';
-  initialState();
-  var shopLists = getLists();
-  var listItems = getListsItems(shopLists[0]);
-  var checked = getListsItems(shopLists[0].concat('.Checked'));
-  parseList(shopLists, addListName);
-  parseList(listItems, addItem);
-  parseChecked(checked);
-  $('.listNames li').each(function() {
-    $(this).removeClass('highlight');
-  });
-  $('.listNames ul').find('input').first().prop('checked', true);
-  $('.listNames ul').find('li').first().toggleClass('highlight', this.checked);
-  $(document).on('mousedown', '.delete', (function(){
-    deleteList.call(this);
-  }));
-  $('#listNameForm').submit(function(e) {
-    var name = $('#listName').val();
-    if (name === '') {
-      $('#listName').addClass('error');
-    } else {
-      $('#listName').removeClass('error');
-      addListName(name);
-      var shopLists = getLists();
-      shopLists.push(name);
-      setLists('Shopping Lists', shopLists);
-      var emptyList = [''];
-      setLists(name.concat('.Checked'), emptyList);
-      $('.listItems ul').empty();
-    }
-    e.preventDefault();
-  });
-  $('#itemNameForm').submit(function(e) {
-    var name = $('#itemName').val();
-    if (name === '') {
-      $('#itemName').addClass('error');
-    } else {
-      $('#itemName').removeClass('error');
-      addItem(name);
-      var listItems = getListsItems($('.listNames ul').find("input:radio:checked").val());
-      if (listItems === null || listItems[0] === '')
```

Comment:
Null and empty string are both falsey, so you could do `if (!listItems [0])` because if listItems is null, listItems[0] will be undefined (false) or, if it's an empty string, it will also be false.